src/view/components/Drawer/index.tsx

```diff
@@ -65,6 +68,17 @@ class Drawer extends React.Component<Props, OwnState> {
     this.props.goToLogin();
   }
 
+  private createDrawerContents() {
+    return (
+      <DrawerContents
+        name={'Joe Trader'}
+        username={'joe@qualitymilk.ca'}
```

Comment:
don't have to change this now but we will probably need to change the name and username. maybe "demo user" and "demouser1"? probably not an email address since we are assuming they don't have an email address

Reply:
Yeah likely, although I think when James sweeps through this it will be changed to the props from the store